### ISSUE TITLE
bottom-sheet: focus trapping does not work unless autoFocus is true

### ISSUE THREAD

**Issue by mmalerba** [angular/material2#15119]
can see the issue on the docs site: https://material.angular.io/components/bottom-sheet/examples

**Comment by mmalerba** [comment]
Currently there are 2 possible workarounds:
1. Set `autoFocus: true` in the bottom sheet config (https://stackblitz.com/edit/angular-mncw9u?file=app/bottom-sheet-overview-example.ts)
2. Manually add your own focus trap (https://stackblitz.com/edit/angular-qv62yz?file=app/bottom-sheet-overview-example-sheet.html)
